metabrainz/model/access_log.py

```diff
@@ -147,3 +147,43 @@ def top_downloaders(cls, limit=None):
         if limit:
             query = query.limit(limit)
         return query.all()
+
+    @classmethod
+    def top_ips(cls, days=7, limit=None):
+        """
+            Generates two list of most active ip addresses in the last days. One
+            list for commercial users and another list for non-commercial users who
+            are not in good standing. good standing for non-commercial users means
+            that they are verified good non-commercial users.
+
+        Args:
+            days: Number of past days to include in the query
+            limit: Max number of items to return.
+
+        Returns:
+            List of <User, request count> pairs
```

Comment:
not quite true? should be a tuple of these pairs with non-commercial first and then commercial